### ISSUE TITLE
Automate end-to-end tests

### ISSUE THREAD

**Issue by D4nte** [comit-network/comit-rs#206]
Write end-to-end tests and run them as part of CI using Node.js.

Note: @bonomat has already done most of the work.

DoD:
* `happy_path.sh` is re-written in Node.js
* Includes some fails scenario such as, but not limited to:
  * Alice does not redeems, Alice & Bob get their funds back
  * Alice tries to redeem after time out
* For BTC-ETH swap (and ETH-LN swap if available)

May depends on #205
